### ISSUE TITLE
Test docker 1.13.1 with new hyperstart and cc-oci-runtime changes 

### ISSUE THREAD

**Issue by chavafg** [01org/cc-oci-runtime#746]
Let's verify if these changes solved the problems in #693, #692 and #691

**Comment by chavafg** [comment]
After latest changes from #753, all tests passed, except swarm, which was expected result.

**Comment by dlespiau** [comment]
Excellent. Should we close this issue then?

**Comment by chavafg** [comment]
Turns out that when running with containerd version 0.2.3, which is the version that comes in Fedora and Ubuntu, there are some issues while killing containers :(

**Comment by chavafg** [comment]
```
fuentess@ubuntu-cc-test:~$ sudo docker run -d ubuntu sleep 5
0f9d2f38dbd74b6b6e4e432dcc8d05335706e81e6f9a8f282c9f9a5d66da1578
fuentess@ubuntu-cc-test:~$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
0f9d2f38dbd7        ubuntu              "sleep 5"           9 seconds ago       Up 8 seconds                            frosty_darwin
fuentess@ubuntu-cc-test:~$ sudo docker kill 0f9d2f38dbd7
0f9d2f38dbd7
fuentess@ubuntu-cc-test:~$ sudo docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
0f9d2f38dbd7        ubuntu              "sleep 5"           43 seconds ago      Up 42 seconds                           frosty_darwin
fuentess@ubuntu-cc-test:~$ docker rm -f 0f9d2f38dbd7
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
fuentess@ubuntu-cc-test:~$ sudo docker rm -f 0f9d2f38dbd7
Error response from daemon: Unable to remove filesystem for 0f9d2f38dbd74b6b6e4e432dcc8d05335706e81e6f9a8f282c9f9a5d66da1578: remove /var/lib/docker/containers/0f9d2f38dbd74b6b6e4e432dcc8d05335706e81e6f9a8f282c9f9a5d66da1578/shm: device or resource busy
```

**Comment by devimc** [comment]
fixed in #776
